Svc/FpySequencer/FpySequencerDirectives.cpp

```diff
@@ -29,10 +29,43 @@ void FpySequencer::sendSignal(Signal signal) {
     }
 }
 
+Fw::Success FpySequencer::sendCmd(FwOpcodeType opcode, const U8* argBuf, FwSizeType argBufSize) {
+    Fw::ComBuffer cmdBuf;
+    Fw::SerializeStatus stat = cmdBuf.serialize(Fw::ComPacketType::FW_PACKET_COMMAND);
```

Comment:
Need to be careful when serializing `Fw::ComPacketType` because the serialization size of `Fw::ComPacketType` is not automatically the same as `FwPacketDescriptorType`. Depending on how the value is deserialized this may cause issues.
The rest of the codebase is doing
```suggestion
    Fw::SerializeStatus stat = cmdBuf.serialize(static_cast<FwPacketDescriptorType>(Fw::ComPacketType::FW_PACKET_COMMAND));
```
We may actually want to put a ticket in to see if we should switch `FwPacketDescriptorType` to be equal to the width of Fw::ComPacketType (which is now U16 because of [this](https://github.com/nasa/fprime/blob/a070ee768aa00d6d433452410a766e3d04f2105c/default/config/ComCfg.fpp#L19C17-L19C20))